RELEASE_NOTES/2.11.0.md

```diff
@@ -0,0 +1,234 @@
+
+#### Key Benefits
+
+- **Synchronous Creation**: Allows for the immediate creation of child datastores without waiting for asynchronous operations.
+- **Strong Typing**: Ensures type safety and better developer experience by leveraging TypeScript's type system.
+
+#### Use Cases
+
+##### Example 1: Creating a Child Datastore
+
+In this example, we demonstrate how to support creating a child datastore synchronously from a parent datastore.
+
+```typescript
+/**
+ * This is the parent DataObject, which is also a datastore. It has a
+ * synchronous method to create child datastores, which could be called
+ * in response to synchronous user input, like a key press.
+ */
+class ParentDataObject extends DataObject {
+  createChild(name: string): ChildDataStore {
+    assert(
+      this.context.createChildDataStore !== undefined,
+      "this.context.createChildDataStore",
+    );
+
+    const { entrypoint } = this.context.createChildDataStore(
+      ChildDataStoreFactory.instance,
+    );
+    const dir = this.root.createSubDirectory("children");
+    dir.set(name, entrypoint.handle);
+    entrypoint.setProperty("childValue", name);
+
+    return entrypoint;
+  }
+
+  getChild(name: string): IFluidHandle<ChildDataStore> | undefined {
+    const dir = this.root.getSubDirectory("children");
+    return dir?.get<IFluidHandle<ChildDataStore>>(name);
+  }
+}
+```
+
+For a complete example see the following test: <https://github.com/microsoft/FluidFramework/blob/main/packages/test/local-server-tests/src/test/synchronousDataStoreCreation.spec.ts>
+
+#### Change details
+
+Commit: [`3426b43`](https://github.com/microsoft/FluidFramework/commit/3426b434dfa06de3ee1a60a5f0d605cd312f2c58)
+
+Affected packages:
+
+- @fluidframework/container-runtime
+- @fluidframework/runtime-definitions
+
+[⬆️ Table of contents](#contents)
+
+### Presence-related events now support the `off` event deregistration pattern ([#23196](https://github.com/microsoft/FluidFramework/issues/23196))
+
+Event subscriptions within `@fluidframework/presence` may now use `off` to deregister event listeners, including initial listeners provided to `Notifications`.
+
+Some type names have shifted within the API though no consumers are expected to be using those types directly. The most visible rename is `NotificationSubscribable` to `NotificationListenable`. Other shifts are to use types now exported through `@fluidframework/core-interfaces` where the most notable is `ISubscribable` that is now `Listenable`.
+
+#### Change details
+
+Commit: [`f7be965`](https://github.com/microsoft/FluidFramework/commit/f7be9651daeba09853627c0953e5969a60674ce3)
+
+Affected packages:
+
+- @fluidframework/presence
+
+[⬆️ Table of contents](#contents)
+
+### Presence updates are now batched and throttled ([#23075](https://github.com/microsoft/FluidFramework/issues/23075))
```

Comment:
Need to use grouping and batching consistently in this section, preferably picking one or the other. I can fix.